app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -32,25 +32,10 @@
     @include core-19;
     color: $white;
     background: $light-blue;
-    padding: ($gutter * 2) $gutter $gutter;
+    padding: 5px $gutter $gutter;
```

Comment:
The top spacing in the screenshot seems to have got smaller?

Reply:
@fofr Yep, this is because the spacing in the title component is different to the spacing of the original title and context that was there before. It doesn't seem like a bad change though?

Reply:
I prefer it to be unchanged.

Reply:
Increasing 5 to 15px would set it back to what it was.